src/app/components/utility/modal/modal.component.ts

```diff
@@ -242,6 +272,42 @@ export class ModalComponent implements OnInit {
       this.isDisabled = e.target.value === this.user.last_name;
     } else if (e.target.name === 'update_affiliation') {
       this.isDisabled = e.target.value === this.user.affiliation;
+    } else if (e.target.name === 'update_google_scholar_url') {
+      if (this.validURL(e.target.value) || e.target.value === '') {
+        this.isDisabled = (e.target.value === '' ? false : e.target.value === this.user.google_scholar_url);
```

Comment:
> > I'm sorry, but I can't reproduce your issue. 

It will fail, if suppose I've entered the different value for 2 fields but now I'm entering the same value for last URL field as `this.user.google_scholar_url` then it will disable the button but it should not happen. Please let me know if I'm not clear.

Reply:
You know what, I'm just gonna set the isDisabled variable as false. It wouldn't be a problem if they submit a duplicate, right?